### PR TITLE
Add NNotepad into index page

### PR DESCRIPTION
**Pull request by ibelem**
Please review if we are ready to add NNotepad into index page.

@huningxin @Honry @inexorabletash

### CODE REVIEW

**Review by Honry — APPROVED** [review]
LGTM, thanks!

**Review by anssiko — APPROVED** [review]
LGTM, awaiting "ship it!" signal from @inexorabletash.

**Comment by inexorabletash** [comment]
FYI I'm on vacation this week, so don't wait for me once the other PR merged. Go go go! :)

**Pull request merged**
